giddy/markov.py

```diff
@@ -282,12 +281,12 @@ def __init__(self, class_ids, classes=None, fill_empty_classes=False, summary=Tr
                     )
                 print(*self.astates_indices, sep=", ")
 
-    @property
-    def mfpt(self):
-        warn("self._mfpt is deprecated. Please use self._mfpt")
-        if not hasattr(self, "_mfpt"):
-            self._mfpt = mfpt(self.p, fill_empty_classes=True)
-        return self._mfpt
+    # @property
```

Comment:
why is this commented out?

Reply:
[`mfpt()` is defined twice there](https://github.com/pysal/giddy/blob/b66ca759f06667bca979c776d5a374f0a0138f75/giddy/markov.py#L285-L296), which triggers a `ruff` error:

```
giddy/markov.py:292:9: F811 Redefinition of unused `mfpt` from line 285
```

Also, I am a bit confused on the need for two identical methods. Should one be removed?

Reply:
Yep, I will make the change and add to the PR. Thank you!